File: Code Chef/JavaScript/700-799/719 Important Pages on CodeChef CHEFPAGES.js
let input = '';
process.stdin.on('data', (data) => input += data);

const decideUrlToSend = (didPractice, didUseContest) => {
    const practicePage = 'https://www.codechef.com/practice';
    const contestPage = 'https://www.codechef.com/contests';
    const discussionPage = 'https://discuss.codechef.com';
    
    const urlToSend = (didPractice && didUseContest)
        ? discussionPage
        : (didPractice && !didUseContest)
            ? contestPage
            : practicePage;

    console.log(urlToSend);
};

process.stdin.on('end', () => {
    const [didPractice, didUseContest] = input.trim().split(' ').map(Number);
    decideUrlToSend(didPractice, didUseContest);
});

File: Code Chef/C Programming/700-799/719 Important Pages on CodeChef CHEFPAGES.c
#include <stdio.h>
int main(void)
{
	int a,b;
	scanf("%d%d",&a,&b);
	if((a==0 && b==0)||(a==0 && b==1))
	{
	    printf("https://www.codechef.com/practice\n");
	}
	else if(a==1 && b==0)
	{
	    printf("https://www.codechef.com/contests\n");
	}
	else
	{
	    printf("https://discuss.codechef.com\n");
	}
	return 0;
}
